### DISCUSSION TITLE
Nextcloud webDAV download from remote into import folder

### DISCUSSION THREAD

**Discussion by kapuett** [photoprism/photoprism#4627] [Q&A]
I have connected my nextcloud instance as webdav service in photoprism. My goal is to regularly sync the remote nextcloud folder with my the `import` folder. Instead it seems to only be possible to automatically sync to `originals` as one cannot set a target in the dialog:
<img width="463" alt="image" src="https://github.com/user-attachments/assets/02251e04-77a0-4be1-a7a9-adb776434480">
Am I missing something here?

**Comment by omerdduran** [comment]
WebDAV sync operation uses the originals folder. Sync with the import folder is not available. But if you want to sync by a third party software, you can connect to the import folder with WebDAV. Like this http://admin@localhost:2342/import/

Important Note: If you are going to sync with a local application, it is recommended that you mount the file storage directory of that application directly as the originals folder instead of using WebDAV. This prevents unnecessary file duplication.
